src/contribute/index.md

```diff
@@ -78,29 +97,90 @@ necessary things to run those tests. For example:
     $ ./gradlew -p sdks/java/io/cassandra check
     $ ./gradlew -p runners/flink check
 
-### Testing the Python SDK
+You can see what build tasks are available with
+
+    $ ./gradlew tasks
+
+or for a module,
+
+    $ ./gradlew sdks/java/io/cassandra tasks
+
+### Developing with an IDE
+
+Generate an IDEA project .ipr file with:
+
+    $ ./gradlew idea
+
+### Pull requests
+
+When your change is ready to be reviewed and merged, create a pull request.
+Format the pull request title like [BEAM-XXX] Fixes bug in ApproximateQuantiles,
```

Comment:
Maybe add backticks or quotes for the title of the Pull Request?

Reply:
done